pkg/secret/secret.go

```diff
@@ -0,0 +1,28 @@
+// Package contains interfaces for API secrets, storage and encoder.
```

Comment:
Documenting a package should follow the convention of [go documentation](https://blog.golang.org/godoc). For example, package documentation should start with `Package [name of the package] ...`